base_tier_validation/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Base Tier Validation",
     "summary": "Implement a validation process based on tiers.",
-    "version": "18.0.1.1.0",
+    "version": "18.0.1.1.1",
```

Comment:
This change is not needed in the PR.
The version number will be automatically changed depending on the merge command.